scheduling/src/main/java/io/helidon/scheduling/Scheduling.java

```diff
@@ -16,29 +16,56 @@
 
 package io.helidon.scheduling;
 
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
+import io.helidon.common.LazyValue;
 import io.helidon.common.configurable.ScheduledThreadPoolSupplier;
 
+import static io.helidon.scheduling.FixedRate.DelayType.SINCE_PREVIOUS_START;
+
 /**
  * Scheduling periodically executed task with specified fixed rate or cron expression.
- *
+ * <p>
+ * Configure a scheduled task using imperative style:
  * <pre>{@code
- * Scheduling.fixedRate()
+ * FixedRate.builder()
  *      .delay(2)
  *      .task(inv -> System.out.println("Executed every 2 seconds"))
  *      .build();
  * }</pre>
  *
  * <pre>{@code
- * Scheduling.cron()
+ * Cron.builder()
  *      .expression("0 45 9 ? * *")
  *      .task(inv -> System.out.println("Executed every day at 9:45"))
  *      .build()
  * }</pre>
+ *
+ * This class also contains nested annotations to use with declarative style (Helidon Declarative, or Helidon MP).
```

Comment:
I don't think we should mention flavors, or keywords like "bean".

We need to find a consistent pattern for documenting the "container/namespace" classes.

E.g.

> Annotations container class that provides declarative style annotations for Helidon Scheduling.